flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -285,6 +286,13 @@ object AkkaUtils {
         |     throughput = $akkaThroughput
         |   }
         | }
+        |
+        | remote {
+        |   netty.tcp {
+        |     maximum-frame-size = $akkaMaximumFrameSize
```

Comment:
does this change imply that `AkkaOptions.FRAMESIZE` is currently broken?

Reply:
I don't think it's broken, just because the akka config returned by this method comes from this configuration template instead of the Configuration object.